### PR TITLE
Lint with rubocop-jekyll

### PR DESCRIPTION
**Pull request by jekyllbot**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -5,18 +5,17 @@ rvm:
   - 2.5
   - 2.4
   - 2.3
-  - 2.2
 matrix:
   include:
     - rvm: 2.4.2
-      env: JEKYLL_VERSION=3.6
+      env: JEKYLL_VERSION=3.7.3
```
Comment (pathawks) [inline .travis.yml]: Because GitHub Pages?

Comment (DirtyF) [reply .travis.yml]: yep

File: jekyll-sass-converter.gemspec [hunk 2]
```diff
@@ -24,5 +24,5 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "jekyll", "~> 3.0"
   spec.add_development_dependency "rake"
   spec.add_development_dependency "rspec"
-  spec.add_development_dependency "rubocop", "~> 0.57.2"
+  spec.add_development_dependency "jekyll-rubocop", "~> 0.1"
```
Comment (pathawks) [inline jekyll-sass-converter.gemspec]: > ```text
> Fetching gem metadata from https://rubygems.org/..
> Could not find gem 'jekyll-rubocop (~> 0.1)' in any of the gem sources listed in your Gemfile.
> ```

**Comment by DirtyF** [comment]
@jekyllbot: merge +dev

**Pull request merged**